### ISSUE TITLE
Not expensive enough to uninstall

### ISSUE THREAD

**Issue by brexitjs** [brexitjs/brexitjs#1]
Need to burn some more £s.

**Comment by adamchainz** [comment]
CPU time is money, it could run heavy loops before uninstalling

**Comment by brexitjs** [comment]
Yeah, maybe as an alternative to https://github.com/brexitjs/brexitjs/pull/3 we could have a while loop instead of a setTimeout. cc @chrisward
